xml/System.Text.Json/Utf8JsonReader.xml

```diff
@@ -651,8 +667,8 @@ For input data within a `ReadOnlySpan<byte>`, this always returns `false`. For i
         <ReturnType>System.Boolean</ReturnType>
       </ReturnValue>
       <Docs>
-        <summary>To be added.</summary>
-        <value>To be added.</value>
+        <summary>Gets the mode of this instance of the <see cref="T:System.Text.Json.Utf8JsonReader" />.</summary>
+        <value><see langword="true" /> if the reader was constructed with the input span containing the entire data to process. <see langword="false" /> if the reader was constructed with an input span that may contain partial data with more data to follow.</value>
```

Comment:
Re-reading this, using the term `span` is incomplete because the reader can be given JSON data contained within a `ReadOnlySequence<byte>` as well.

Maybe:
```suggestion
        <value><see langword="true" /> if the reader was constructed with the input span or sequence containing the entire JSON data to process. <see langword="false" /> if the reader was constructed with an input span or sequence that may contain partial JSON data with more data to follow.</value>
```